SR2025/2024-10-05-mentoring.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2025 teams
+subject: Mentoring 
```

Comment:
I've been trying to ensure we include the year in subject lines when doing generic emails, so that they're distinguishable year to year.
Perhaps:
```suggestion
subject: Student Robotics 2025 Mentoring
```